vignettes/faq.Rmd

```diff
@@ -126,6 +126,13 @@ a way to look up common ADaM variables with recommended `{admiral}` functions to
   * _"It is a huge advantage for all Pharma companies that we have common functions for common stuff we develop. It will be easier for the authorities when it is the same foundation in the ADaM programs. The development goes faster for every one when we develop across companies, and bug-fixing is faster as many are using same package and will most likely find potential bugs."_
   * _"I am a huge proponent of shared solutions within Pharma. Overall I was VERY impressed with the `{admiral}` project -- both the design, development, documentation, and validation details are available for teams to readily adopt."_
 
+##### Do `{admiral}` functions **preserve variable attributes** like labels?
+
+* Most `{admiral}` derivation functions rely on `{dplyr}` operations, which generally preserve attributes on existing columns but do not automatically transfer attributes to newly created columns.
+* Admiral utility functions like `convert_blanks_to_na()` and `convert_na_to_blanks()` are explicitly designed to preserve all attributes.
```

Comment:
```suggestion
* Most `{admiral}` functions don't preserve variable attributes because they rely on `{dplyr}` operations, which generally don't preserve attributes.
* For admiral functions which preserve attributes like `convert_blanks_to_na()` and `convert_na_to_blanks()` this is explicitly mentioned in the documentation.
```